compiler/extccomp.nim

```diff
@@ -645,7 +645,7 @@ proc compileCFile(conf: ConfigRef; list: CFileList, script: var Rope, cmds: var
     if optCompileOnly notin conf.globalOptions:
       add(cmds, compileCmd)
       let (_, name, _) = splitFile(it.cname)
-      add(prettyCmds, "CC: " & name)
+      add(prettyCmds, if hintCC in conf.notes: "CC: " & name else: "")
```

Comment:
Instead of adding empty strings into ``prettyCmds`` wouldn't it be cleaner to move the if statement up?

Reply:
Ha, I said the same but it was ignored.

Reply:
ya, that's how i had initially implemented it until it failed because of that. Anyway, ok to merge? tests have succeeded already